compiler/onecc-docker/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved 
+# 
+# Licensed under the Apache License, Version 2.0 (the "License"); 
+# you may not use this file except in compliance with the License. 
+# You may obtain a copy of the License at 
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0 
+# 
+# Unless required by applicable law or agreed to in writing, software 
+# distributed under the License is distributed on an "AS IS" BASIS, 
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+# See the License for the specific language governing permissions and 
+# limitations under the License 
+
+FROM ubuntu:18.04 
```

Comment:
Q) @jyoungyun , if we are to prepare for Ubuntu 20.04, or other versions, how are we going to name the Docker file?

Reply:
> Q) @jyoungyun , if we are to prepare for Ubuntu 20.04, or other versions, how are we going to name the Docker file?

This docker file parameters the Debian package version using the '--build-arg' option.  How about parameterizing Ubuntu version like Debian package version?

Reply:
> Q) @jyoungyun , if we are to prepare for Ubuntu 20.04, or other versions, how are we going to name the Docker file?

I agree your opinion.  :)

@holawan

We are going to official release for Ubuntu 20.04. After Ubuntu 20.04 is released, we will provide a docker image for Ubuntu 20.04 too. In this case, if the `Dockerfile` name is used as it is, it overlaps.

What about changing the `Dockerfile` name and use `-f` option when you build docker image?

Reply:
I think we can go as-is for now, and revise this later :)

Reply:
> I think we can go as-is for now, and revise this later :)

@holawan

Let's go as-is. :)

Reply:
> What about changing the Dockerfile name and use -f option when you build docker image?

Is it okay to create a Dockerfile with two different names? (ex: Dockerfile_20_04, Dockerfile_18.04)
Or, would you like to create two directories and place the Dockerfile in each folder?